codex-ui/src/vue/components/alert/Alert.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <transition name="fadeInBottom">
+    <div
+      v-show="message"
+      class="alert"
+      :class="`alert--${alertType}`"
+    >
+      <Icon name="Check" />
+      <span v-if="props.message">{{ message }}</span>
+      <slot />
+    </div>
+  </transition>
+</template>
+
+<script setup lang="ts">
+import { defineProps, computed, withDefaults } from 'vue';
+import Icon from '../icon/Icon.vue';
+
+const props = withDefaults(
+  defineProps<{
+  /**
+   * message to display
+   */
+    message?: string;
+
+    /**
+     * alert status
+     */
+    type: 'success' | 'error' | 'warning' | 'default' | 'info';
+
+  }>(), {
+    message: '',
+    type: 'default',
+  }
+);
+
+/** determine alert status */
+const alertType = computed(() => {
+  if (props.type === 'success') {
+    return 'success';
+  }
+
+  if (props.type === 'error') {
+    return 'error';
+  }
+
+  if (props.type === 'warning') {
+    return 'warning';
+  }
+
+  if (props.type === 'info') {
+    return 'info';
+  }
+
+  return 'default';
+});
+
+</script>
+
+<style  lang="postcss">
+
+@keyframes fadeInBottom {
+  0% {
+    transform: translateY(50px);
+    opacity: 0;
+  }
+
+  100% {
+    transform: translateY(0);
+    opacity: 1;
+  }
+}
+
+.alert {
+  --padding-left: var(--h-padding);
+  --padding-right: var(--h-padding);
+  --success: #00A64D;
+  --error: #BB393D;
+  --warning: #DE6205;
+  --info: #54617B;
+  --default: #54617B;
```

Comment:
use `theme-base="grass"`, `theme-base="sky"` and `data-base="red"` and `--base--bg-primary` for bg and `--base--text` for color